Scripts/Miscellaneous/Connect4Game/requirements.txt

```diff
@@ -0,0 +1,5 @@
+Before starting the game make sure to install pygame package into your local system or virtual environment
```

Comment:
requirements.txt file must include only this:

numpy==1.19.2
pygame==1.9.6

Reply:
Updated the requirements.txt file as per as you requested.